app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @recipe.name %></h1>
+<p>Recipe: <%= @recipe.name %></p>
```

Comment:
I would change `Recipe:` to `Name:` and do the same for complexity and genre. So:
```ruby
<p>Name: <%= @recipe.name %></p>
<p>Complexity: <%= @recipe.complexity %></p>
<p>Genre: <%= @recipe.genre %></p>
```